cmd/zdb/zdb.c

```diff
@@ -6042,12 +6118,27 @@ main(int argc, char **argv)
 	argc -= optind;
 	argv += optind;
 
-	if (argc < 2 && dump_opt['R'])
+	if (getenv("ZFS_OPTIONS_DEBUG") != NULL) {
```

Comment:
Can you either document `ZFS_OPTIONS_DEBUG` in the `zdb` man page, or if it's intended to only be used for internal ZTS testing add a comment to that effect here.

Reply:
it is for testing only, I'll add references to make it clearer